scripts/r.in.aster/r.in.aster.py

```diff
@@ -166,7 +183,10 @@ def import_aster(proj, srcfile, tempfile, output, band):
     grass.debug("gdalwarp -t_srs %s %s %s" % (proj, srcfile, tempfile))
 
     if platform.system() == "Darwin":
```

Comment:
I suspect this architecture detection for Mac dates back to FAT binary era. But even then I don't understand why this had to be explicitly added to the command.

May I suggest to try removing lines 185-190 and out-indent the line with `cmd = ["gdalwarp", "-t_srs", proj, srcfile, tempfile]` (which will be the only remaining one).

Reply:
hei @nilason! that's a good point.  I have been conservative, as I expected it to be something related also with M1/arm which I can not test (intel machine here).   

from the arch manual:

```     
     ...
     The other use of the arch command is to run a selected architecture of a
     universal binary.  A universal binary contains code that can run on
     different architectures.  By default, the operating system will select
     the architecture that most closely matches the processor type.  A 64-bit
     architecture is preferred over a 32-bit architecture on a 64-bit
     processor, while only 32-bit architectures can run on a 32-bit processor.

     When the most natural architecture is unavailable, the operating system
     will try to pick another architecture.  On 64-bit processors, a 32-bit
     architecture is tried.  Otherwise, no architecture is run, and an error
     results.
     ...
```

I suppose gdal for OSX is _universal binary_ , and don't know about the FAT binary era.  So it seem safe to simplify with the solution you suggest!

Reply:
An ARM machine would need "arm64", eventually the list will be long...  :)

Reply:
Got it!  I took some time to study the historical context of the [fat binary](https://en.wikipedia.org/wiki/Fat_binary).  

As far as I understand, nowadays we have gdal compiled natively for the platform we are running GRASS in.   So I'd go with your suggested code, adding a comment about this #3258 to keep track of this discussion for future reference.

Reply:
It _could_ be a universal build (eg. x86_64+arm64), but it should work either way. You can safely remove the comment, the history will be in git!